Unable to install package via npx expo install

### Description

Duplicates #124 

```bash
npm ERR! code 1
npm ERR! path C:\Users\...\node_modules\react-native-esc-pos-printer
npm ERR! command failed
npm ERR! command C:\WINDOWS\system32\cmd.exe /d /s /c node ./scripts/movelib.js
npm ERR! node:internal/modules/cjs/loader:1147
npm ERR!   throw err;
npm ERR!   ^
npm ERR! 
npm ERR! Error: Cannot find module 'fs-extra'
npm ERR! Require stack:
npm ERR! - C:\Users\...\node_modules\react-native-esc-pos-printer\scripts\movelib.js
npm ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:1144:15)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:985:27)
npm ERR!     at Module.require (node:internal/modules/cjs/loader:1235:19)
npm ERR!     at require (node:internal/modules/helpers:176:18)
npm ERR!     at Object.<anonymous> (C:\Users\...\node_modules\react-native-esc-pos-printer\scripts\movelib.js:1:12)
npm ERR!     at Module._compile (node:internal/modules/cjs/loader:1376:14)
npm ERR!     at Module._extensions..js (node:internal/modules/cjs/loader:1435:10)
npm ERR!     at Module.load (node:internal/modules/cjs/loader:1207:32)
npm ERR!     at Module._load (node:internal/modules/cjs/loader:1023:12)
npm ERR!     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:135:12) {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: [
npm ERR!     'C:\\Users\\...\\node_modules\\react-native-esc-pos-printer\\scripts\\movelib.js'
npm ERR!   ]
npm ERR! }
npm ERR! 
npm ERR! Node.js v20.11.0
```
### Steps to reproduce

1. npx expo install react-native-esc-pos-printer

### react-native-esc-pos-printer version

4.4.1

### React Native version

0.79.2

### Platforms

Android, iOS

### Workflow

Expo Go

### Architecture

None

### Build type

None

### Device

None

### Device model

_No response_

### Acknowledgements

Yes

--- a/package/scripts/movelib.js
+++ b/package/scripts/movelib.js
@@ -1,41 +1,66 @@
-const fs = require('fs-extra');
+const fs = require('fs');
 const path = require('path');
-const fsNative = require('fs');
 
 let dir = '../../react-native-esc-pos-printer-sdk';
 
-if (!fsNative.existsSync(path.join(__dirname, dir))) {
+if (!fs.existsSync(path.join(__dirname, dir))) {
   dir = '../node_modules/react-native-esc-pos-printer-sdk';
 }
 
 const libCurrentPath = path.join(__dirname, dir, 'ios');
-
 const libDestPath = path.join(__dirname, '../ios/PrinterSDK');
 
-// copy iOS sdk
-fs.copy(libCurrentPath, libDestPath, (err) => {
-  if (err) return console.error(err);
-  console.log('iOS SDK copied!');
-});
-
 const libAndroidCurrentPath = path.join(__dirname, dir, 'android/libs');
-
 const libAndroidDestPath = path.join(__dirname, '../android/libs');
 
 const jnilibAndroidCurrentPath = path.join(__dirname, dir, 'android/jniLibs');
-
 const jnilibAndroidDestPath = path.join(
   __dirname,
   '../android/src/main/jniLibs'
 );
 
-// copy Android sdk
-fs.copy(libAndroidCurrentPath, libAndroidDestPath, (err) => {
-  if (err) return console.error(err);
-  console.log('Android lib copied!');
-});
+// Helper to recursively copy directory
+function copyDirSync(src, dest) {
+  if (!fs.existsSync(src)) return;
 
-fs.copy(jnilibAndroidCurrentPath, jnilibAndroidDestPath, (err) => {
-  if (err) return console.error(err);
+  if (!fs.existsSync(dest)) {
+    fs.mkdirSync(dest, { recursive: true });
+  }
+
+  const entries = fs.readdirSync(src, { withFileTypes: true });
+
+  for (const entry of entries) {
+    const srcPath = path.join(src, entry.name);
+    const destPath = path.join(dest, entry.name);
+
+    if (entry.isDirectory()) {
+      copyDirSync(srcPath, destPath);
+    } else {
+      fs.copyFileSync(srcPath, destPath);
+    }
+  }
+}
+
+// Copy iOS SDK
+try {
+  copyDirSync(libCurrentPath, libDestPath);
+  console.log('iOS SDK copied!');
+} catch (err) {
+  console.error(err);
+}
+
+// Copy Android libs
+try {
+  copyDirSync(libAndroidCurrentPath, libAndroidDestPath);
+  console.log('Android lib copied!');
+} catch (err) {
+  console.error(err);
+}
+
+// Copy Android jniLibs
+try {
+  copyDirSync(jnilibAndroidCurrentPath, jnilibAndroidDestPath);
   console.log('Android jniLibs copied!');
-});
+} catch (err) {
+  console.error(err);
+}

